ext/MitsubaLoader/CGlobalMitsubaMetadata.h

```diff
@@ -56,6 +59,8 @@ class IMeshMetadata : public IMitsubaMetadata
 		struct Instance
 		{
 			core::matrix3x4SIMD tform;
+			SContext::bsdf_type bsdf;
+			std::string bsdf_id;
```

Comment:
why do you need the string?

Reply:
for debug

Reply:
ok maybe only have that field in DEBUG builds

Reply:
TODO